--- php-pear-1.9.4/tests/AllTests.php~	2012-03-23 07:23:21.790166417 +0100
+++ php-pear-1.9.4/tests/AllTests.php	2012-03-23 07:23:25.169171549 +0100
@@ -46,9 +46,7 @@ if (!defined('PHPUnit_MAIN_METHOD')) {
 /*
  * Files needed by PhpUnit
  */
-require_once 'PHPUnit/Framework.php';
-require_once 'PHPUnit/TextUI/TestRunner.php';
-require_once 'PHPUnit/Extensions/PhptTestSuite.php';
+require_once 'PHPUnit/Autoload.php';
 
 /*
  * You must add each additional class-level test suite file here
--- php-pear-1.9.4/tests/testCase/BasicGraph.php~	2012-03-23 07:23:15.515156885 +0100
+++ php-pear-1.9.4/tests/testCase/BasicGraph.php	2012-03-23 07:23:27.490175072 +0100
@@ -25,7 +25,7 @@
 //
 
 require_once 'Structures/Graph.php';
-require_once 'PHPUnit/Framework.php';
+require_once 'PHPUnit/Autoload.php';
 
 /**
  * @access private
